### PR TITLE
Fix websocket trying to reconnect after a logout

### PR DESCRIPTION
**Pull request by jwilander**
Also fixed copy/paste bug in signing up a new team

### CODE REVIEW

File: webapp/action_creators/websocket_actions.jsx [hunk 1]
```diff
@@ -147,6 +152,7 @@ export function sendMessage(msg) {
 }
 
 export function close() {
+    manuallyClosed = true;
```
Comment (coreyhulen) [inline webapp/action_creators/websocket_actions.jsx]: does this work?  I think close also get called after onerror?

Comment (jwilander) [inline webapp/action_creators/websocket_actions.jsx]: It's called in the unmount of logged_in.jsx https://github.com/mattermost/platform/blob/master/webapp/components/logged_in.jsx#L177 which will occur on logout and also when the window unloads https://github.com/mattermost/platform/blob/master/webapp/root.jsx#L110

**Comment by coreyhulen** [comment]
+1

**Comment by crspeller** [comment]
+1

**Pull request merged**